mne_bids/mne_bids.py

```diff
@@ -533,8 +606,16 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     config : str | None
         A path to the configuration file to use if the data is from a BTi
         system.
-    overwrite : bool
-        If the file already exists, whether to overwrite it.
+    write_mode : str, one of ('overwrite', 'error', 'clear')
+        How any existing files should be handled when new data is produced.
+        Defaults to 'overwrite'.
+        - 'overwrite': Any file that already exists will be overridden with new
+            data, except `participants.tsv` and `scans.tsv` which will have the
+            new data appended intelligently.
```

Comment:
remove "intelligently" ? It's confusing :)

Reply:
+1